src/org/rascalmpl/library/lang/rascal/tests/library/IO.rsc

```diff
@@ -25,3 +26,16 @@ test bool createdDoesNotCrashOnURIRewrites() {
     writeFile(|tmp:///createdDoesNotCrashOnURIRewrites/someFile.txt|, "123456789");
     return created(|tmp:///createdDoesNotCrashOnURIRewrites/someFile.txt|) <= now();
 }
+
+test bool testWriteBase32() {
+    str original = "Hello World!";
+    writeBase32(|tmp:///base32Test/writeTest.txt|, toBase32(original));
```

Comment:
for (new) tests we tend to use the `memory:///` uri, to avoid writing to disk during the test.

Reply:
Much better!